application/src/main/java/org/ehrbase/application/config/web/WebConfiguration.java

```diff
@@ -72,14 +87,17 @@ public void addCorsMappings(CorsRegistry registry) {
     public void addInterceptors(@NonNull InterceptorRegistry registry) {
         if (auditContext.isAuditEnabled()) {
             // Composition endpoint
-            registry.addInterceptor(new CompositionAuditInterceptor(auditContext, ehrService, compositionService))
-                    .addPathPatterns("/rest/openehr/v1/**/composition/**");
+            registry.addInterceptor(new CompositionAuditInterceptor(
+                            auditContext, ehrService, compositionService, tenantService))
+                    .addPathPatterns(COMPOSITION_PATH_PATTERN.formatted(apiContextPath, EHR, COMPOSITION));
```

Comment:
Just an idea, but if we had constants 
`ANY_SEGMENT = "*"`
`ANY_TRAILING_SEGMENTS = "**"`
and a method
``` java
String contextPathPattern(String ... segments) {
    return Stream.concat(Stream.of(apiContextPath), Arrays.stream(segments)).collect(Collectors.joining("/");
}
```
we could just do
```java
.addPathPatterns(contextPathPattern(EHR, ANY_SEGMENT, COMPOSITION, ANY_TRAILING_SEGMENTS))
```

Reply:
done